source/extensions/filters/common/lua/wrappers.cc

```diff
@@ -67,7 +67,9 @@ int BufferWrapper::luaGetBytes(lua_State* state) {
 
 int BufferWrapper::luaSetBytes(lua_State* state) {
   data_.drain(data_.length());
-  absl::string_view bytes = luaL_checkstring(state, 2);
+  size_t input_size;
+  const char* input = luaL_checklstring(state, 2, &input_size);
+  absl::string_view bytes = absl::string_view(input, input_size);
```

Comment:
May be we can wrapper these code into a function? Because we repeat these code in multiple positions?

Reply:
We also need some new test case to cover these new codes.

Reply:
ok